TESTING.md

```diff
@@ -63,12 +63,47 @@ tests are finished. The command below will give you a shell after all tests are
 
 ```
 # cd keylime-tests
-# tmt run -vvv prepare discover provision -h virtual -i Fedora-35 execute login finish
+# tmt run -vvv prepare discover provision -h virtual -i Fedora-35 -c system execute login finish
 ```
 
 You can use it to inspect test logs or even modify test sources and run your tests
 manually following the method described above. For this purpose you can find test sources under `/var/tmp/tmt/run-*/packit-ci/discover/default/tests/`.
 
+### Running multi-host tests
+
+`tmt` cannot schedule multi-host tests yet. However, we can use `tmt` to do the necessary setup and then execute the test manually.
+
+For `Multihost/basic-attestation` test we need at least 3 test systems that
+will be started using `tmt`.
+
+Let's start with opening 3 terminals on your workstation.
+
+On terminal 1:
+```
+$ git clone https://github.com/RedHat-SP-Security/keylime-tests.git
+```
+
+On all terminals:
+```
+$ cd keylime-tests
+$ tmt run -vvv prepare discover -h fmf -t 'configure_tpm_emulator' -t 'install_upstream_keylime' -t 'Multihost/basic-attestation' provision -h virtual -i Fedora-35 execute login finish
```

Comment:
Perhaps it might be useful to mention that during this step it will download an image for using in the test? And actually, that it might be wise to wait for the image to be downloaded before going to the 2nd and 3rd terminals to provision the other systems, as the image can be reused once it's downloaded.

In my test, it was a Fedora-Cloud-Base-35-1.2.x86_64.qcow2 file, and I don't know how it works, but apparently it was getting it from a really slow mirror, so it was taking very long, this download. I actually ended up stopping it and then downloading it myself from a different mirror, once I found it was expected to be at `/var/tmp/tmt/testcloud/images`.

Another point that might be useful to mention is that we need libvirtd running, or elsewe will get errors like 
```
progress: booting...
libvirt: XML-RPC error : Failed to connect socket to '/var/run/libvirt/virtqemud-sock': No such file or directory
libguestfs: error: could not connect to libvirt (URI = qemu:///system): Failed to connect socket to '/var/run/libvirt/virtqemud-sock': No such file or directory [code=38 int1=2]
```

Another issue I had the first time (this was a "brand new" minimal Fedora 35 system) was with the tmt provision stuff. I needed some specific package to be able to use tmt provision virtual; package probably is named tmt-virtual, if I recall correctly. I found out there was a tmt-all and installed this one instead, though.

Reply:
Hi @sergio-correia , thank you for a detailed review. I have updated doc with additional instructions. Also, please note that the top level section 'Running tests in a virtual system using tmt' contains more details regarding the required workstation setup.

Reply:
Yeah, now I see that I missed that section. Thanks!